clients/admin-ui/src/features/dataset/EditCollectionOrFieldForm.tsx

```diff
@@ -41,19 +41,21 @@ const EditCollectionOrFieldForm = ({ values, onSubmit, dataType }: Props) => {
     data_qualifier: values.data_qualifier,
     data_categories: values.data_categories,
   };
-  const allDataCategories = useSelector(selectDataCategories);
+  const allEnabledDataCategories = useSelector(selectDataCategories).filter(
```

Comment:
it might be slightly nicer to have a separate selector in the slice file called something like `selectEnabledDataCategories` which looks something like:

```typescript
export const selectEnabledDataCategories = createSelector(selectDataCategories, (dataCategories) =>
  dataCategories.filter(dc => dc.active)
);
```
and then just call `selectEnabledDataCategories` wherever you need it (instead of having the component call a filter each time).

this really isn't a big deal right now though, so you can consider it optional!